service/informer/informer.go

```diff
@@ -232,22 +230,32 @@ func containsEmptyItems(projects []string) bool {
 }
 
 func ensureProject(projects []draughtsmantprspec.Project, project draughtsmantprspec.Project) []draughtsmantprspec.Project {
-	var updated bool
```

Comment:
I rewrote this a bit and wrote tests for it. We considered the ID being an identifier. This is not true. The name is the identifier.